### PR TITLE
Fix: update the remaining kubecf.suse.dev links

### PR DESCRIPTION
**Pull request by ikapelyukhin**
## Description

Point all of the remaining kubecf.suse.dev links to kubecf.io, as the old site is currently broken.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: chart/README.md [hunk 1]
```diff
@@ -49,5 +49,5 @@ instructions.
 [3]: https://github.com/cloudfoundry-incubator/kubecf/blob/master/doc/Contribute.md#customization
 [4]: https://quarks.suse.dev/docs/quarks-operator/concepts/variables/#explicit-variables
 [5]: https://quarks.suse.dev/docs/quarks-operator/concepts/variables/#implicit-variables
-[6]: https://kubecf.suse.dev/docs/getting-started/kubernetes-deploy/
-[7]: https://kubecf.suse.dev/docs/
+[6]: https://kubecf.io/docs/getting-started/kubernetes-deploy/
```
Comment (mook-as) [inline chart/README.md:52]: This link is broken; perhaps we want https://kubecf.io/docs/deployment/ here? (Pretty sure it was broken before the domain switch, too.)

File: chart/values.yaml [hunk 2]
```diff
@@ -371,7 +371,7 @@ features:
       resource_directory_key: ~
 
   # Configuration for the external database; see also features.embedded_database.  Please refer to
-  # https://kubecf.suse.dev/docs/getting-started/kubernetes-deploy/#external-database for details.
+  # https://kubecf.io/docs/getting-started/kubernetes-deploy/#external-database for details.
```
Comment (mook-as) [inline chart/values.yaml:374]: This should go to https://kubecf.io/docs/deployment/kubernetes-deploy/#external-database probably

**Review by mook-as — CHANGES_REQUESTED** [review]
Two (already-)broken links.

File: chart/README.md (hunk 1 above)
Comment (ikapelyukhin) [reply chart/README.md:52]: Looks like it moved to https://kubecf.io/docs/deployment/kubernetes-deploy/.

Comment (ikapelyukhin) [reply chart/README.md]: Fixed.

File: chart/values.yaml (hunk 2 above)
Comment (ikapelyukhin) [reply chart/values.yaml]: Yep, fixed.

**Review by mook-as — APPROVED** [review]
Thanks for the changes!

**Pull request merged**